module/VuFind/src/VuFind/Record/FallbackLoader/Summon.php

```diff
@@ -62,7 +62,7 @@ class Summon extends AbstractFallbackLoader
      */
     protected function fetchSingleRecord($id)
     {
-        $resource = $this->table->findResource($id, 'Summon');
+        $resource = $this->resourceService->getResourceByRecordId($id, 'Summon');
```

Comment:
It was actually a bug that this was previously creating resource rows when looking up IDs.